flink-core/src/test/java/org/apache/flink/core/io/LocatableSplitAssignerTest.java

```diff
@@ -428,7 +428,7 @@ void testConcurrentSplitAssignmentForMultipleHosts() throws InterruptedException
         assertThat(ia.getNextInputSplit("testhost", 0)).isNull();
 
         // at least one fraction of hosts needs be local, no matter how bad the thread races
-        assertThat(ia.getNumberOfRemoteAssignments())
+        assertThat(ia.getNumberOfLocalAssignments())
```

Comment:
Do not include commits that don't belong to this PR.

Reply:
done